Source/MonoGame.Extended.NuclexGui/Controls/Desktop/GuiResizableControl.cs

```diff
@@ -0,0 +1,247 @@
+﻿using MonoGame.Extended.Input.InputListeners;
+using MonoGame.Extended;
+using System;
+using Microsoft.Xna.Framework.Input;
+
+namespace MonoGame.Extended.NuclexGui.Controls.Desktop
+{
+    [Flags]
+    public enum ResizingSides
+    {
+        None = 0,
+        Top = 1,
+        Right = 2,
+        TopRight = 3,
```

Comment:
The sequence here should not be with powers of two? Like 0, 1, 2, 4, 8, 16 and so on?

Reply:
Those represent combinations of flags, Top Right is Top & Right.

This allows you to say `resizingDirection.HasFlag(TopRight) if you'd like, instead of "resizingDirection.HasFlag(Top) && resizingDirection.HasFlag(Right)".  

According to this: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum

"CONSIDER providing special enum values for commonly used combinations of flags."

That's what those represent.

Reply:
You're right, I did not know this guideline. Thank you for the knowledge.